app/components/nav-bar.js

```diff
@@ -8,5 +8,21 @@ export default Component.extend({
       this.get('authManager').logout();
       this.get('routing').transitionTo('index');
     }
+  },
+
+  didInsertElement() {
+    this._super.call(this);
+    this.$('.notification.item').popup({
+      popup : '.popup',
+      on    : 'click'
+    });
+  },
+
+  willDestroyElement() {
+    this._super.call(this);
+    this.$('.notification.item').popup({
```

Comment:
You need to destroy the popup here. Not create it. See semantic ui docs on how to destroy a popup.